permission-portal-frontend/client/src/components/ChangePasswordModal.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Modal from '../components/Modal'
+import PendingOperationButton from '../components/PendingOperationButton'
+
+class ChangePasswordModal extends React.Component {
```

Comment:
This boilerplate code is different from other components we have.

Reply:
@fordacious what do you mean by that? As in its a class component rather than a function component?